xctemplates/Router Templates/Router.xctemplate/___FILEBASENAME___Router.swift

```diff
@@ -4,28 +4,16 @@ import Foundation
 import SwiftUI
 
 class ___FILEBASENAMEASIDENTIFIER___: Router {
-    
-    // MARK: - Published vars
-    // Put published vars here
-    
-    // MARK: - Private vars
-    
-    // MARK: - Internal vars
     var services: Services
-        
-    // MARK: - Initialization
 
     init(services: Services) {
         self.services = services
     }
-    
-    // MARK: - Methods
```

Comment:
Why did you remove this pragma marks?

Reply:
Pragma marks are typically used at the file level to help explain sections of your code.  Adding them _within_ a class is also a valid use case but typically only done once those sections are large enough to not fit on screen and are needed to help navigate.  Inner class pragma marks are also a sign that your class may be too large and you could consider breaking it up in a different way to avoid this.  Including pragma marks by default at the class level before knowing if you really need them or considering alternatives just seemed like something we could avoid.

Reply:
You make a good point there.

How can we enforce code ordering without them? Shall we add a code guideline? Comments that are not marks?

Reply:
Yeah it will have to be done through code review and we could certainly add structure into guidelines as something that will be looked for.

Reply:
I'll add that